libs/coin-modules/coin-evm/src/logic/getSequence.ts

```diff
@@ -1,6 +1,7 @@
 import { CryptoCurrency } from "@ledgerhq/types-cryptoassets";
 import { getNodeApi } from "../network/node";
 
-export function getSequence(currency: CryptoCurrency, address: string): Promise<number> {
-  return getNodeApi(currency).getTransactionCount(currency, address);
+export async function getSequence(currency: CryptoCurrency, address: string): Promise<bigint> {
+  const txCount = await getNodeApi(currency).getTransactionCount(currency, address);
+  return BigInt(txCount?.toString() || 0);
```

Comment:
You should be able to simply use the number here
```suggestion
  return BigInt(txCount);
```